docs/buildah-mount.md

```diff
@@ -6,7 +6,7 @@ buildah\-mount - Mount a working container's root filesystem.
 ## SYNOPSIS
 **buildah** **mount**
 
-**buildah** **mount** **containerID**
+**buildah** **mount** [**containerID** [...]]
```

Comment:
one small nitpick though, should it be `[containerID [...]]` or `[containerID ...]`? Because in `mount.go` you specified `ArgsUsage:      "[CONTAINER-NAME-OR-ID, ...]",`.

is this consistent?

Reply:
This is good here, but the the usage does need to change please.